src/screens/GroupInfo/index.js

```diff
@@ -181,7 +186,7 @@ const GroupInfo = () => {
         <>
           <Header isCenter onPress={() => navigation.goBack()} title={memberName()} />
           <View style={styles.lineTop} />
-          <ScrollView nestedScrollEnabled={true}>
+          <ScrollView contentContainerStyle={styles.scrollContainer} nestedScrollEnabled={true}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `contentContainerStyle` prop has been added to the `ScrollView` component. This is generally used to apply styles to the inner container which wraps all children. Just ensure that the styles applied here are necessary and do not conflict with other styles in child components.
```diff
- <ScrollView contentContainerStyle={styles.scrollContainer} nestedScrollEnabled={true}>
+ <ScrollView contentContainerStyle={styles.scrollContainer} nestedScrollEnabled={true}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->